ShopifySharp/Entities/FulfillmentServiceEntity.cs

```diff
@@ -0,0 +1,88 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// An object representing a Shopify fulfillment service.
+    /// </summary>
+    public class FulfillmentServiceEntity : ShopifyObject
```

Comment:
Rename this class to `FulfillmentService` rather than `FulfillmentServiceEntity`.

Reply:
Also the file should be `FulfillmentService.cs` once renamed, sorry!

Reply:
That's originally what I wanted to name it, but because Entities and Services aren't in their own namespace, there will be a conflict with the Service already named FulfillmentService.

Reply:
Yeah, I didn't like adding the suffix either, but didn't see any way around it.